test/snapshots/BuildTest.js.snap

```diff
@@ -4853,6 +4859,54 @@ Object {
 exports[`posix smart unpack 4`] = `
 Array [
   "app.asar",
+  "app.asar.unpacked/node_modules/keytar/LICENSE.md",
+  Object {
+    "content": "{
+  \\"main\\": \\"./lib/keytar.js\\",
+  \\"typings\\": \\"keytar.d.ts\\",
+  \\"name\\": \\"keytar\\",
+  \\"description\\": \\"Bindings to native Mac/Linux/Windows password APIs\\",
+  \\"version\\": \\"7.9.0\\",
+  \\"license\\": \\"MIT\\",
+  \\"repository\\": {
+    \\"type\\": \\"git\\",
+    \\"url\\": \\"https://github.com/atom/node-keytar.git\\"
+  },
+  \\"homepage\\": \\"http://atom.github.io/node-keytar\\",
+  \\"files\\": [
+    \\"lib\\",
+    \\"src\\",
+    \\"binding.gyp\\",
+    \\"keytar.d.ts\\"
+  ],
+  \\"types\\": \\"./keytar.d.ts\\",
+  \\"devDependencies\\": {
+    \\"babel-core\\": \\"^6.26.3\\",
+    \\"babel-plugin-transform-async-to-generator\\": \\"^6.24.1\\",
+    \\"chai\\": \\"^4.2.0\\",
+    \\"mocha\\": \\"^9.2.0\\",
+    \\"node-cpplint\\": \\"~0.4.0\\",
+    \\"node-gyp\\": \\"^8.4.1\\",
+    \\"prebuild\\": \\"^11.0.2\\"
+  },
+  \\"dependencies\\": {
+    \\"node-addon-api\\": \\"^4.3.0\\",
+    \\"prebuild-install\\": \\"^7.0.1\\"
+  },
+  \\"binary\\": {
+    \\"napi_versions\\": [
+      3
+    ]
+  },
+  \\"config\\": {
+    \\"runtime\\": \\"napi\\",
+    \\"target\\": 3
+  }
+}",
+    "name": "app.asar.unpacked/node_modules/keytar/package.json",
+  },
+  "app.asar.unpacked/node_modules/keytar/lib/keytar.js",
```

Comment:
Why are the `LICENSE.md`, `package.json`, and `keytar.js` showing up in the unpacked folder?

Reply:
In smartunpack, if an executable file is detected, the entire folder containing it will automatically be placed into app.asar.unpack, not just the file itself.

https://github.com/electron-userland/electron-builder/blob/84f290990915d7ffa0d2dfe0a1d739aeb2b21560/packages/app-builder-lib/src/asar/unpackDetector.ts#L29